### PR TITLE
extend reuseMeta to vectorized input; adresses #20

### PR DESCRIPTION
**Pull request by beckerbenj**
Extends `reuseMeta()` to work with input vectors (multiple variables instead of just one).

### CODE REVIEW

File: R/reuseMeta.R [hunk 1]
```diff
@@ -21,15 +21,38 @@
...
+  if(inherits(other_GADSdat, "GADSdat")) check_GADSdat(other_GADSdat)
+  if(inherits(other_GADSdat, "all_GADSdat")) check_all_GADSdat(other_GADSdat)
   if(is.null(other_varName)) other_varName <- varName
+  check_vars_in_GADSdat(other_GADSdat, vars = other_varName)
+  if(length(varName) != length(other_varName)) {
+    stop("'varName' and 'other_varName' have different length.")
+  }
+
+  if(!is.null(missingLabels) && !missingLabels %in% c("drop", "leave", "only")) {
+    stop("Invalid input for argument missingLabels.")
```
Comment (franikowsp) [inline R/reuseMeta.R:42]: Nit: Maybe, add: `"Must be either \"drop\", \"leave\", or \"only\"`?

File: R/reuseMeta.R [hunk 2]
```diff
@@ -21,15 +21,38 @@
...
+reuseMeta.GADSdat <- function(GADSdat, varName, other_GADSdat, other_varName = NULL,
+                              missingLabels = NULL, addValueLabels = FALSE) {
+  check_GADSdat(GADSdat)
+  check_vars_in_GADSdat(GADSdat, vars = varName)
+  if(inherits(other_GADSdat, "GADSdat")) check_GADSdat(other_GADSdat)
+  if(inherits(other_GADSdat, "all_GADSdat")) check_all_GADSdat(other_GADSdat)
   if(is.null(other_varName)) other_varName <- varName
+  check_vars_in_GADSdat(other_GADSdat, vars = other_varName)
+  if(length(varName) != length(other_varName)) {
+    stop("'varName' and 'other_varName' have different length.")
```
Comment (franikowsp) [inline R/reuseMeta.R:38]: Nit: "different length**s**"?

**Review by franikowsp — APPROVED** [review]
Looks all good to me.

File: R/reuseMeta.R (hunk 2 above)
Comment (beckerbenj) [reply R/reuseMeta.R:38]: Mh, it was a conscious choice, similar to this:
https://english.stackexchange.com/questions/7820/of-different-size-or-of-different-sizes

Do you think it sounds very 'wrong'?

Comment (franikowsp) [reply R/reuseMeta.R:38]: Sorry, I did not do the research - does not sound wrong, but I was not sure about this. But the explanation makes sense to me! 👌

File: R/reuseMeta.R (hunk 1 above)
Comment (beckerbenj) [reply R/reuseMeta.R:42]: agree and added!

**Comment by beckerbenj** [comment]
Thanks as always for the super quick and helpful review @franikowsp !

**Pull request merged**